src/routers/submit.router.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response, Router, NextFunction } from "express";
+import { env } from "../config";
+
+const router: Router = Router();
+const { SUBMIT_VALIDATION_URL, FILE_ID_FROM_ACCOUNT_VALIDATOR_CALLBACK } = env;
+router.get('/submit', (_req: Request, res: Response, _next: NextFunction) => {
+    res.redirect(`${SUBMIT_VALIDATION_URL}?callback=${FILE_ID_FROM_ACCOUNT_VALIDATOR_CALLBACK}`);
```

Comment:
For the callback URL we should just be configuring the hostname and not the path.
The path should be configured by a constant that can be used for both the router and this callback.
This will ensure they always match.
```suggestion
    const zipPortalBaseURL = // CHS_URL or custom URL if ZIP portal is not hosted in CHS.
    res.redirect(`${SUBMIT_VALIDATION_URL}?callback=${zipPortalBaseURL}${uploadedEndpointPath}/{fileId}`);
```